test/controllers/tasks_controller_test.rb

```diff
@@ -24,6 +24,14 @@ def in_taxonomy_scope(organization, location = nil)
         Organization.current = Location.current = nil
       end
 
+      it 'supports csv export' do
```

Comment:
This should have been done from the beginning of the feature: we would have found the issue much faster than without it.